sys/fuchsia/fidlgen/main.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+var zirconLibs = []string{
+	"fuchsia-process",
+	"fuchsia-io",
+}
+
+func main() {
+	targetArch := os.ExpandEnv("${TARGETARCH}")
```

Comment:
`os.Getenv("TARGETARCH")` is simpler, use it

Reply:
Done